src/eduid_common/session/session_cookie.py

```diff
@@ -112,6 +112,11 @@ def _verify_session_id(cls, session_id: bytes, signing_key: bytes, signature: by
 
         return bool(not invalid_bits)
 
+    def derive_key(self, app_secret: str, usage: str, size: int) -> bytes:
```

Comment:
isn't this confusing with the derive_key below in the outer scope?

Reply:
At this time, I didn't want to declare derive_key to be _only_ related to SessionCookie, but I wanted to encapsulate the complexity of juggling the types of session_id into SessionCookie. Good or bad I do not know.